internal/appsec/config/rules_manager.go

```diff
@@ -46,18 +46,6 @@ type (
 	RulesData struct {
 		RulesData []RuleDataEntry `json:"rules_data"`
 	}
-
-	// ActionEntry represents an entry in the "actions" top level field of a rules file
-	ActionEntry struct {
```

Comment:
do you recall why did we typedef'ed it in the past?

Reply:
Yup, that was back when we needed to parse the actions provided in the ruleset in order to keep track of them in the WAF handle.
The WAF used to return an action ID only and we had to track the mapping from ID->action.
Now the WAF returns the whole action definition so we don't need to track them, and to parse them out from the ruleset.